internal/buildscripts/packaging/installer/install.sh

```diff
@@ -386,12 +389,26 @@ install() {
         install_yum_package "td-agent" "$td_agent_version"
         systemctl stop td-agent
       fi
+      if [ -f "/opt/td-agent/bin/fluent-cap-ctl" -a command -v yum >/dev/null 2>&1; ]; then
+        yum group install "Developement Tools"
```

Comment:
Need to also support `dnf` for systems without `yum`.

Reply:
We should be able to run `dnf group install "Development Tools"` if `yum` is not installed, similar to what the `install_yum_package` function does. I have it in https://github.com/signalfx/splunk-otel-collector/pull/139.